3723/3/app/controllers/place_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PlaceController < ApplicationController
+  post '/places/:id' do
+    @comment = Comment.new(
```

Comment:
Use `Comment.create` instead of `Comment.new` then will be unnecessary usage of `@comment.save`